backend/wmg/pipeline/summary_cubes/expression_summary/job.py

```diff
@@ -83,7 +92,15 @@ def create_expression_summary_cube(corpus_path: str, default=False):
         gene_ontology_term_ids = extract_var_data(corpus_path, ctx)
 
         # transform
-        cube_index, cube_sum, cube_nnz = transform(corpus_path, gene_ontology_term_ids, cube_dims)
-        _load(uri, gene_ontology_term_ids, cube_index, cube_sum, cube_nnz, default=default)
+        result = transform(corpus_path, gene_ontology_term_ids, cube_dims)
```

Comment:
I think the args here need to be keyword arguments. The pipeline failed with an error indicating as such:
https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logsV2:log-groups/log-group/%2Fdp%2Frdev%2Fstack-name%2Fwmg-processing/log-events/wmg-rdev-stack-name-wmg-processing%2Fdefault%2F1bd4a7129c9d4943a9881f7974f6389b